examples/trace-analytics-sample-app/sample-app/analytics-service/build.gradle

```diff
@@ -27,9 +27,9 @@ repositories {
 dependencies {
     implementation('org.springframework.boot:spring-boot-starter-web')
     testImplementation('org.springframework.boot:spring-boot-starter-test')
-    implementation 'org.elasticsearch.client:elasticsearch-rest-high-level-client:7.8.1'
-    implementation 'org.elasticsearch.client:elasticsearch-rest-client:7.8.1'
-    implementation 'org.elasticsearch:elasticsearch:7.8.1'
+    implementation 'org.opensearch.client:opensearch-rest-high-level-client:1.1.0'
+    implementation 'org.opensearch.client:opensearch-rest-client:1.1.0'
+    implementation 'org.opensearch:opensearch:1.1.0'
```

Comment:
I think we are not even using these dependency in the analytics-service sample app now. Those depedency was for the legacy logic in loggingController: https://github.com/chenqi0805/simple-ingest-transformation-utility-pipeline-1/blob/enh/kibana-trace-analytics-plugin/examples/odfe-pipes-trace-app/analytics-service/src/main/java/com/example/restservice/LoggingController.java
which has been significantly simplified.

We could either remove them in this PR or in later PR. Also you might want to pay attention to application.properties in the resources:

```
server.port=8087
elasticsearch.host=elasticsearch
elasticsearch.port=9200
```

elasticsearch.host=elasticsearch, elasticsearch.port=9200 can both be removed

Reply:
@dlvenable Can you check if these are quick fixes? If so, let's fix them in this PR.

Reply:
Yes, I'll fix them in this PR. Thanks for catching this.

I think we are not using it as well, but I wanted to remove these dependencies from our whole build.gradle dependency chain.

Reply:
Thanks @chenqi0805 , I have removed those dependencies and the properties from the application.properties file.